### DISCUSSION TITLE
GitHub Education Application for the University of Stuttgart

### DISCUSSION THREAD

**Discussion by luerhard** [community/community#53283] [GitHub Education]
Dear GitHub Team,
I just tried to apply for GitHub Education as a teacher and have problems getting my Email-Adress recognized. 

My University: University of Stuttgart (https://www.uni-stuttgart.de)

# Problem

The error message always is: 

```
We require applicants to use a school-issued email address to apply.
```
I did not get the chance to proof my affiliation by uploading documents.

# Possible Reasons

I found several problems that could be the reason for this:

## 1. My university uses department-specific subdomains

I am affiliated with the social science department, so my Email-Adress ends on `@sowi.uni-stuttgart.de` 

## 2. The institution is shown multiple times in your dropdown menu

- Once as "Unversity of Stuttgart (https://www.uni-stuttgart.de)" which is correct but there are no valid email-domains shown

- Once as "Universitaet Stuttgart (https://www.uni-stuttgart.de/en)" which is the German translation of the name. Here, the assigned domain of `gs-gp.eu`  is NOT of the University of Stuttgart but of a vocational school around 45km east of Stuttgart.

**Comment by samus-aran** [comment]
Hi @luerhard  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Comment by luerhard** [comment]
Great. The support told me to go here....

![grafik](https://user-images.githubusercontent.com/24412608/235475425-771649ea-9696-4b15-8b42-9ded306388b4.png)
